### DISCUSSION TITLE
[ FF7 ] Crash after demon wall battle - temple of ancients - 60FPS

### DISCUSSION THREAD

**Discussion by phoenixskye** [julianxhokaxhiu/FFNx#357] [General]
Has anyone else had their game crash at the temple of the ancients after completing the demon wall boss fight? I recreated the crash several times, crashing at the same moment each time when Cait Sith enters the room. I progressed past the temple as soon as I disabled 60fps.

**Comment by tangtang95** [comment]
Do you have a save file just before the boss fight for me to try?

**Reply by phoenixskye** [reply]
I didn’t think to do that I’m sorry :( I saved over it as soon as I got past the temple. No issues since though.

**Comment by Sewje** [comment]
Updating mods, graphics drivers, FFNx combination will fix this. dont need to disable 60fps mod.
